### ISSUE TITLE
Change Subtitle Font and Position (Desktop Client)

### ISSUE THREAD

**Issue by stewieboo** [Stremio/stremio-features#611]
Windows 11 Pro (**Build 22631**)
Stremio Desktop (**Ver. 4.4.164**)

**Feature request:** 
I am not sure which font is currently being used by the desktop client, but I'd like to have the option to change it to something else (e.g., Netflix Sans, Clear Sans Bold, etc.). Also, if I could change the font position vertically, it would be helpful. If adding a font changing feature doesn't seem viable, please consider at least allowing the user to change its vertical position, such as the one on the web version of Stremio. Thank you so much for reading. ^^

**Comment by MikeKaiser55** [comment]
Yes Pls! Its one of the main reasons i use stremio web and i wish could be implemented in the desktop version too!

**Comment by sisi399** [comment]
I'd gladly contribute $100 to whoever adds the feature of adjusting subtitle vertical position via shortcut keys (like ALT+UP/DOWN) to the Desktop client.

**Comment by Avtera** [comment]
when this will available for pc?

**Comment by TRtomasz** [comment]
new stremio shell 5 allows changing subtitles position

**Comment by ufuksarp** [comment]
Please re-open this issue. I just refuse to read ugly Arial font.
